pkg/controller/applicationmonitoring/templateHelper.go

```diff
@@ -187,6 +188,20 @@ func newTemplateHelper(cr *applicationmonitoring.ApplicationMonitoring, extraPar
 	}
 }
 
+func SkipGrafanaServiceAccount() string {
+	value, exist := os.LookupEnv("LOCAL_INSTALL")
+	if exist {
+		// If local install is true then do not skip service account creation
+		if value == "True" {
+			return "False"
+		} else {
+			return "True"
+		}
+	}
+
+	return "False"
```

Comment:
btw: in the version of operatorSDK that is used here currently, there is an env var that can be used to check if it's running locally, see:
https://github.com/integr8ly/application-monitoring-operator/blob/master/vendor/github.com/operator-framework/operator-sdk/pkg/k8sutil/k8sutil.go#L183

Reply:
I just realized that if we rely on the env var that I reference in the comment above, it will break whenever newer operator-sdk is used, so ignore this.
There are better ways to check if it's running locally, e.g.:
https://github.com/integr8ly/integreatly-operator/blob/master/pkg/resources/k8s/k8sutil.go#L26-L39
but for now, we can leave it as is.